Guessing Game/GuessingGame.js

```diff
@@ -0,0 +1,24 @@
+// *Challenge
+/* 
+Write a JavaScript program where the program takes a random integer between 1 to 10,
+the user is then prompted to input a guess number. If the user input matches with guess number,
+the program will display a message "Good Work!" otherwise display a message "Not matched! The correct number was x", where 'x' 
+should be the guess number. */
+
+// *Tips
+
+/* Write your code in the solution.js file.
+To run run/test your program, simply open/refresh the index.html file in a browser, since solution.js is included as a script in the html file's body.
+To prompt the user use prompt('');
+To notify the user about the result use alert('');
+*/
+
+function getUserNumber() {
+  const promptUser = prompt("Insert a random number between 1 and 10.");
+  const randomNumber = Math.floor(Math.random() * 10);
+  if (parseInt(promptUser) === randomNumber) {
+    alert("Bravo pi4");
+  } else {
+    alert("Proval");
```

Comment:
It would be better for the UX to show what was the random number.

You can show the random number using a template literal (template string  ` `` `  ).